rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -151,15 +151,15 @@ public double minerMinFeesNotifyInDollars() {
     }
 
     public boolean bloomServiceEnabled() {
-        return getBoolean("bloom.service", false);
+        return getBoolean("blooms.service", false);
```

Comment:
isn't there a test we could make to ensure the property is loaded correctly? like loading some fake system property and then getting it from these methods (maybe it's not worth it - but we didn't notice this wasn't working in the first place)

Reply:
good point. Done.